pages/public_cloud/ai_machine_learning/training_tuto_09_train_model_export_onnx/guide.en-gb.md

```diff
@@ -0,0 +1,379 @@
+if __name__ == '__main__':
+
+    train_loader, test_loader = load_data()   
+
+    device =  check_gpu()
+    model = Network().to(device)
+    optimizer = optim.SGD(model.parameters(), lr=0.01, momentum=0.5)
+
+    train_losses, train_accu, test_losses, test_accu = ([] for i in range(4))
+
+    for epoch in range(0, 10):
+        print('\nEpoch %d/%d:'%(epoch, 10))
+        train_model(model, device, train_loader, optimizer, epoch)
+        test_model(model, device, test_loader)
+
+    torch.save(model.state_dict(),"/workspace/models/model_mnist_classification.pt")
+    export_onnx(model)
+```
+
+> [!primary]
+>
+> Find the **full Python code** on our [GitHub repository]https://github.com/ovh/ai-training-examples/blob/main/jobs/onnx/train_image_classification.py).
+>
+
+
+### Create the requirements.txt file
+
+Then, create a [requirements.txt](https://github.com/ovh/ai-training-examples/blob/main/jobs/onnx/requirements.txt) file to declare the Python dependencies and their versions.
+
+```bash
+torch==2.0.1
+torchvision==0.15.2
+onnx==1.14.1
+```
+
+### Build your own Docker image
+
+#### Create a Dockerfile compliant with AI Training
+
+The [Dockerfile](https://github.com/ovh/ai-training-examples/blob/main/jobs/onnx/Dockerfile) should start with the `FROM` instruction indicating the parent image to use. In our case we choose to start from a `python:3.10` image.
+
+Then, precise the workspace path, install the Python dependencies and launch the model training using the `CMD`.
+
+```docker
+FROM python:3.10
+
+WORKDIR /workspace
+ADD . /workspace
+
+RUN pip install -r requirements.txt
+
+CMD [ "python3"  , "/workspace/train_image_classification.py"]
+
+RUN chown -R 42420:42420 /workspace
+ENV HOME=/workspace
+```
+
+#### Build the Docker image from the Dockerfile
+
+Launch the following command from the **Dockerfile** directory to build your application image:
+
+```console
+docker build . -t train-cnn-model-export-onnx:latest
+```
+
+> [!primary]
+>
+> The dot `.` argument indicates that your build context (place of the **Dockerfile** and other needed files) is the current directory.
+>
+> The `-t` argument allows you to choose the identifier to give to your image. Usually image identifiers are composed of a **name** and a **version tag** `<name>:<version>`. For this example we chose **train-cnn-model-export-onnx:latest**.
+>
+
+> [!warning]
+>
+> Please make sure that the docker image you will push in order to run containers using AI products respects the **linux/AMD64** target architecture. You could, for instance, build your image using **buildx** as follows:
+>
+> `docker buildx build --platform linux/amd64 ...`
+>
+
+#### Push the image into the shared registry
+
+> [!warning]
+>
+> The shared registry of AI Deploy should only be used for testing purposes. Please consider attaching your own docker registry. More information about this can be found [here](/pages/public_cloud/ai_machine_learning/training_guide_05_howto_add_registry).
```

Comment:
```suggestion
> The shared registry of AI Deploy should only be used for testing purposes. Please consider attaching your own docker registry. More information about this can be found [here](/pages/public_cloud/ai_machine_learning/gi_07_manage_registry).
```